launcher/minecraft/launch/LauncherPartLaunch.cpp

```diff
@@ -159,6 +161,113 @@ void LauncherPartLaunch::executeTask()
+        bwrapArgs << "--dev" << "/dev";
+        bwrapArgs << "--dev-bind-try" << "/dev/dri" << "/dev/dri";
+        bwrapArgs << "--proc" << "/proc";
+        bwrapArgs << "--setenv" << "XDG_RUNTIME_DIR" << sandboxedRuntimeDir;
+
+        for (auto path : systemBinds) {
+            bwrapArgs << "--ro-bind-try" << path << path;
+        }
+
+        // desktop integration
+        bwrapArgs << "--ro-bind-try" << QString("%1/pulse").arg(actualRuntimeDir) << QString("%1/pulse").arg(sandboxedRuntimeDir);
+        bwrapArgs << "--ro-bind-try" << QString("%1/pipewire-0").arg(actualRuntimeDir) << QString("%1/pipewire-0").arg(sandboxedRuntimeDir);
+        {
+            auto display = qEnvironmentVariable("DISPLAY");
+            auto xAuthPath = qEnvironmentVariable("XAUTHORITY",
+                    FS::PathCombine(QStandardPaths::writableLocation(QStandardPaths::HomeLocation), ".Xauthority"));
+            auto wlDisplay = qEnvironmentVariable("WAYLAND_DISPLAY");
+
+            if (display.startsWith(':')) {
+                auto x11Socket = QString("/tmp/.X11-unix/X%1").arg(display.mid(1));
+                bwrapArgs << "--ro-bind-try" << x11Socket << x11Socket;
+            }
+
+            bwrapArgs << "--ro-bind-try" << xAuthPath << xAuthPath;
+
+            if (wlDisplay.startsWith('/'))
+                bwrapArgs << "--ro-bind-try" << wlDisplay << wlDisplay;
+            else
+                bwrapArgs << "--ro-bind-try" << FS::PathCombine(actualRuntimeDir, wlDisplay) << FS::PathCombine(sandboxedRuntimeDir, wlDisplay);
+        }
+
+        if (minecraftInstance->settings()->get("EnableMangoHud").toBool() && APPLICATION->capabilities() & Application::SupportsMangoHud) {
+            auto mangoHudConfigPath = qEnvironmentVariable("MANGOHUD_CONFIGFILE",
+                    FS::PathCombine(QStandardPaths::writableLocation(QStandardPaths::GenericConfigLocation), "MangoHud")); 
+            bwrapArgs << "--ro-bind-try" << mangoHudConfigPath << mangoHudConfigPath;
+        }
+
+        // launcher
+        {
+            QString instPath = QDir::toNativeSeparators(QDir(minecraftInstance->instanceRoot()).absolutePath());
+            QString assetsPath = QDir::toNativeSeparators(QDir("assets").absolutePath());
+            QString librariesPath = QDir::toNativeSeparators(QDir("libraries").absolutePath());
+
+            bwrapArgs << "--bind" << instPath << instPath;
+            bwrapArgs << "--ro-bind" << assetsPath << assetsPath;
+            bwrapArgs << "--ro-bind" << librariesPath << librariesPath;
+
+            // also add NewLaunch.jar just to be safe. This is probably covered by /usr or LINUX_BWRAP_EXTRA_ARGS though
+            bwrapArgs << "--ro-bind" << jarPath << jarPath;
+        }
+
+        bwrapArgs << Commandline::splitArgs(BuildConfig.LINUX_BWRAP_EXTRA_ARGS);
+        // TODO: add args from environment variable?
+        bwrapArgs << Commandline::splitArgs(minecraftInstance->settings()->get("BwrapExtraArgs").toString());
+
```

Comment:
```suggestion

bwrapArgs << "--setenv" << "PRISM_INSTANCE_SANDBOXED" << "1";
```
This would help other programs determine if the sandbox is enabled.